src/styles/main.scss

```diff
@@ -1,3 +1,334 @@
+@import "./utils/variables";
```

Comment:
Why do you have two main.scss files?
![image](https://user-images.githubusercontent.com/96409563/212877874-4dac6bd8-c386-497f-8267-20da839262dd.png)